### PR TITLE
Add for review

### PR DESCRIPTION
**Pull request by LukasReschke**
@nickvergessen Have fun.

### CODE REVIEW

**Comment by LukasReschke** [comment]
@tomneedham At some point you need to explain that magic generation script we have to me :see_no_evil:

**Comment by LukasReschke** [comment]
These are for the 7.0.5, 6.0.7 and 5.0.19 release.

File: server/oc-sa-2015-001.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+         "CVE":"TBD",
+         "Operator":"<"
+      },
+      {
+         "Version":"5.0.19",
+         "CVE":"TBD",
+         "Operator":"<"
+      }       
+   ],
+   "Description":"<p>Due to not sanitising all user provided input, the \"contacts\" application shipped with the mentioned ownCloud versions is vulnerable to multiple stored cross-site scripting attacks.<br/>The \"contacts\" application is enabled by default in the ownCloud Community Edition but not shipped with the ownCloud Enterprise Edition.</p><p>Successful exploitation requires that the adversary is to able to access the contact group and able to share contacts with the victim. The victim then has to access the contacts application and edit the maliciously drafted contact.</p><p>While ownCloud advises browsers to disable inline JavaScript execution this vulnerability is caused by a <code>eval</code> like construct which is currently allowed in our default Content-Security-Policy, thus this is effectively exploitable in any browser.</p>",
```
Comment (nickvergessen) [inline server/oc-sa-2015-001.json]: is ~~to~~ able to access

Comment (nickvergessen) [inline server/oc-sa-2015-001.json]: and ~~able to~~ share

File: server/oc-sa-2015-002.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+         "CVE":"TBD",
+         "Operator":"<"
+      },
+      {
+         "Version":"5.0.19",
+         "CVE":"TBD",
+         "Operator":"<"
+      }       
+   ],
+   "Description":"<p>Due to not sanitising all user provided input, the \"documents\" application shipped with the mentioned ownCloud versions is vulnerable to multiple stored cross-site scripting attacks.<br/>The \"documents\" application is enabled by default in the ownCloud Community Edition but not shipped with the ownCloud Enterprise Edition.</p><p>Successful exploitation requires that the adversary is to able to modify a WebODF document and a victim opens the shared document.</p><p>ownCloud advises browsers to disable inline JavaScript execution due to the used Content-Security-Policy; this vulnerability is therefore not exploitable if you use a browser that supports the current CSP standard. You can check at <a href=\"http://caniuse.com/#feat=contentsecuritypolicy\">CanIUse.com</a> whether your browser supports our Content-Security-Policy.</p>",
```
Comment (nickvergessen) [inline server/oc-sa-2015-002.json]: adversary is ~~to~~ able to modify

File: server/oc-sa-2015-003.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+         "CVE":"TBD",
+         "Operator":"<"
+      },
+      {
+         "Version":"5.0.19",
+         "CVE":"TBD",
+         "Operator":"<"
+      }       
+   ],
+   "Description":"<p>A blacklist bypass vulnerability including UTF-8 encoding in file paths in the mentioned ownCloud versions, when running on a Microsoft Windows Platform, allows authenticated remote attackers to bypass the file blacklist and upload files such as the <code>.htaccess</code> files.</p><p>An attacker could leverage this bypass by uploading a <code>.htaccess</code> and execute arbitrary PHP code if the <code>/data/</code> directory is stored inside the webroot and a webserver that interprets <code>.htaccess</code> files is used (e.g. Apache)</p><p>ownCloud always recommends to move the <code>data</code> directory outside of the web root.</p>",
```
Comment (nickvergessen) [inline server/oc-sa-2015-003.json]: should mention server when talking about windows, to make clear that this is a server issue

**Comment by tomneedham** [comment]
:+1:

**Pull request merged**